test/e2e_node/container_lifecycle_test.go

```diff
@@ -731,6 +733,146 @@ var _ = SIGDescribe("[NodeConformance] Containers Lifecycle ", func() {
 	})
 })
 
+var _ = SIGDescribe("[Serial] Containers Lifecycle ", func() {
+	f := framework.NewDefaultFramework("containers-lifecycle-test-serial")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
+
+	ginkgo.It("should restart the containers in right order after the node reboot", func(ctx context.Context) {
```

Comment:
Is this test case assuming the feature gate is disabled? Would it pass with the feature gate enabled?

Reply:
No it doesn't. We'll fix the case with feature gate enabled in #120269 after this.